src/app/home/home.component.html

```diff
@@ -3,36 +3,36 @@
         <img src="../../assets/CashmereLogo.svg" height="190px" width="auto" alt="Fabric.Cashmere Logo"/>
     </span>
     <span class="headerRight">
-        <img src="../../assets/FabricCashmere.svg" height="55px" width="auto" alt="Fabric.Cashmere Title"/>
+        <img src="../../assets/FabricCashmere.svg" height="100%" width="auto" alt="Fabric.Cashmere Title"/>
```

Comment:
Looks like most inline styles were ripped out :+1: I'd recommend throwing a class or id on these last few images and divs and pulling out the height/width stuff, too

Reply:
Yep, had to take them out so I could override them in the media queries.  Good suggestion on moving the height/width on all the imgs to classes.  I'll get an updated commit pushed this afternoon.

Reply:
I would love if something like this would be a global style, as responsive images are common.  I am working with jaque on this stuff so I may do that

Reply:
That's an excellent idea.  I know Bootstrap has a responsive image class, and that would be awesome if we could do something similar.  I found that Firefox and Chrome handled % img dimensions a bit differently, so the class might take a bit of tweaking.  But to your point, that's probably outside of the scope of this PR - maybe open an issue for it if you get a minute and assign yourself, me and Jacquie for starters.